### PR TITLE
permissions.yml delsinrowe

### PR DESCRIPTION
**Pull request by adambash**

### CODE REVIEW

File: PermissionsEx/permissions.yml [hunk 1]
```diff
@@ -8383,6 +8383,38 @@ groups:
...
+        - ch.alias.run0
+        - ch.alias.run1
+        - ch.alias.run2
+        - ch.alias.run3
+        - ch.alias.run4
+        - ch.alias.run5
+        - ch.alias.run6
+        - ch.alias.run7
+        - ch.alias.run8
+        - ch.alias.run9
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: proof of super speed required

Comment (adambash) [inline PermissionsEx/permissions.yml]: http://infamous.wikia.com/wiki/File:InFAMOUS-Second-Son-Delsin-Running-Neon-Afterimages-001.jpg

File: PermissionsEx/permissions.yml [hunk 2]
```diff
@@ -8383,6 +8383,34 @@ groups:
...
+        - ch.alias.jump2
+        - ch.alias.jump3
+        - ch.alias.jump4
+        - ch.alias.jump5
+        - ch.alias.run0
+        - ch.alias.run1
+        - ch.alias.run2
+        - ch.alias.run3
+        - ch.alias.intangible
+        - ch.alias.tangible
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: proof of intangibility?

Comment (adambash) [inline PermissionsEx/permissions.yml]: http://www.dailymotion.com/video/x52s2t4

go to minute 8:40

Comment (malonnnn) [inline PermissionsEx/permissions.yml]: denied, he wasnt going through walls, he was in a smoke form that passed through a grate.

**Pull request merged**